packages/params/test/e2e/ensure-config-is-synced.test.ts

```diff
@@ -10,11 +10,14 @@ import {loadConfigYaml} from "../yaml";
 async function downloadRemoteConfig(preset: "mainnet" | "minimal", commit: string): Promise<Record<string, unknown>> {
   const phase0Url = `https://raw.githubusercontent.com/ethereum/eth2.0-specs/${commit}/presets/${preset}/phase0.yaml`;
   const altairUrl = `https://raw.githubusercontent.com/ethereum/eth2.0-specs/${commit}/presets/${preset}/altair.yaml`;
+  const mergeUrl = `https://raw.githubusercontent.com/ethereum/eth2.0-specs/${commit}/presets/${preset}/merge.yaml`;
```

Comment:
Can you generalize this with types so we don't forget on the next fork? For example:
```ts
const urlByFork: Record<ForkName, string> = {
  [ForkName.phase0]: ...
  ...
}
```